error-codes.md

```diff
@@ -13,6 +13,12 @@ This document describes the problems encountered during the use of TiDB and prov
 TiDB is compatible with the error codes in MySQL, and in most cases returns the same error code as MySQL. For a list of error codes for MySQL, see [Server Error Message Reference](https://dev.mysql.com/doc/refman/5.7/en/server-error-reference.html).
 In addition, TiDB has the following unique error codes:
 
+> **Note:**
+>
+> Some errors are internal errors. Normally, TiDB handles the errors rather than return to the user. So it is not listed here.
+>
+> If you encounter the error that is not listed here, contact PingCAP for support or seek help in the official forum.
```

Comment:
```suggestion
> If you encounter an error code that is not listed here, [contact PingCAP](mailto:info@pingcap.com) for support.
```
The official forum is AskTUG (in Chinese only), so we can remove this information in English docs.